src/dotnet-interactive-vscode/src/interactiveNotebook.ts

```diff
@@ -1,7 +1,22 @@
 import { RawNotebookCell } from "./interfaces";
 import { trimTrailingCarriageReturn } from './utilities';
 
-export const editorLanguages = ['csharp', 'fsharp', 'html', 'javascript', 'markdown', 'powershell'];
+let languageAliases = new Map<string, string>();
```

Comment:
Are these intended to be for code coloring / language service selection on the VS Code side, or for subkernel routing on the .NET Interactive side?

If the latter, we don't currently support `cs` and `fs` but maybe we should.

Reply:
These are to normalize `cs` to `csharp`, etc., so the kernel should never see `cs`, but I agree, adding `cs`/`c#`/`C#`/etc. would be good.

Reply:
We support `C#` and `c#` currently as kernel name aliases.